### DISCUSSION TITLE
Error occurred when generating search queries: 'Docs' object has no attribute 'generate_search_query'

### DISCUSSION THREAD

**Discussion by JannikSchneider12** [Future-House/paper-qa#178] [General]
Hey,

I wanted to test this app (https://huggingface.co/spaces/lifan0127/zotero-qa) and tried to run it locally because I want to replace the openai model with an open llm. In the web it works but if I run the app locally, I get this error: Error occurred when generating search queries: 'Docs' object has no attribute 'generate_search_query'

Does anyone know how I can fix this?

Thanks for your time and help

**Comment by jamesbraza** [comment]
Hi @JannikSchneider12, that space is outside of our control, feel free to open a discussion there: https://huggingface.co/spaces/lifan0127/zotero-qa/discussions

Also, we have moved to version 5, which is a major rewrite and not backwards compatible. If you're still hitting issues with version 5, feel free to open a separate issue
